src/backends/torch/torchmodule.cc

```diff
@@ -122,8 +122,9 @@ namespace dd
         _logger->info("loading " + tmodel._native);
         try
           {
+            // partial weights loading is allowed only if finetuning
             torch_utils::load_weights(*_native, tmodel._native, _device,
-                                      _logger);
+                                      _logger, !_finetuning);
```

Comment:
here

Reply:
OK, let's add a comment, e.g. `// finetuning relaxes strictness when loading weights
Otherwise I can see the case where it's hard to track where `strict` is set.

Reply:
Done